definitions/marts/looker_studio_marts/ecf/ls_ecf_correct_cohort_checker.sqlx

```diff
@@ -0,0 +1,255 @@
+    ,pro.cohort AS current_cohort
+    ,pro.induction_start_date
+    ,pro.eligible_for_funding
+    ,pro.partnership_id
+    ,pro.partnership_relationship
+    ,dec.earliest_declaration_date AS earliest_declaration_date
+    ,CASE
+      WHEN induction_start_date < '2021-09-01' THEN '2023'
+      WHEN induction_start_date BETWEEN '2021-09-01' AND '2022-05-31' THEN '2021'
+      WHEN induction_start_date BETWEEN '2022-06-01' AND '2022-08-31' THEN '2021,2022'
+      WHEN induction_start_date BETWEEN '2022-09-01' AND '2023-05-31' THEN '2022'
+      WHEN induction_start_date BETWEEN '2023-06-01' AND '2023-08-31' THEN '2022,2023'
+      WHEN induction_start_date BETWEEN '2023-09-01' AND '2024-05-31' THEN '2023'
+      ELSE NULL
+    END AS start_date_suggested_true_cohort
+    ,CASE
+      WHEN earliest_declaration_date BETWEEN '2021-09-01' AND '2023-05-31' THEN '2021,2022'
+      WHEN earliest_declaration_date BETWEEN '2023-06-01' AND '2023-08-31' THEN '2021,2022,2023'
+      WHEN earliest_declaration_date BETWEEN '2023-09-01' AND '2024-05-31' THEN '2023'
+     END AS earliest_dec_suggested_true_cohort
+    ,CASE
+      WHEN pro.partnership_id IS NOT NULL AND pro.partnership_relationship = FALSE AND dec.lead_provider_name = 'National Institute of Teaching' THEN 1
+      ELSE 0
+     END AS niot_partnership_chk
+    ,ROW_NUMBER() OVER(PARTITION BY pro.participant_profile_id ORDER BY pro.participant_profile_id ASC) AS participant_precedence
+  FROM 
+    golden_thread pro
+  LEFT JOIN 
+    declarations dec 
+  ON
+    pro.participant_profile_id = dec.participant_profile_id
+  WHERE
+    pro.participant_role IS NOT NULL
+  ORDER BY
+    pro.user_id ASC, pro.participant_profile_id ASC, pro.participant_role ASC, dec.declaration_date ASC
+),
+
+-- Build preliminary output data which includes the binary result for cohort matching each of the suggested values.
+output_data AS (
+  SELECT 
+    *
+    ,CASE
+      WHEN start_date_suggested_true_cohort = '2023' AND current_cohort = 2023 THEN 1
+      WHEN start_date_suggested_true_cohort = '2022' AND current_cohort = 2022 THEN 1
+      WHEN start_date_suggested_true_cohort = '2021' AND current_cohort = 2021 THEN 1
+      WHEN start_date_suggested_true_cohort = '2022,2023' AND current_cohort IN (2022, 2023) THEN 1
+      WHEN start_date_suggested_true_cohort = '2021,2022' AND current_cohort IN (2021, 2022) THEN 1
+      ELSE 0
+     END AS start_date_suggestion_match
+    ,CASE
+      WHEN earliest_dec_suggested_true_cohort = '2023' AND current_cohort = 2023 THEN 1
+      WHEN earliest_dec_suggested_true_cohort = '2021,2022,2023' AND current_cohort IN (2021,2022,2023) THEN 1
+      WHEN earliest_dec_suggested_true_cohort = '2021,2022' AND current_cohort IN (2021,2022) THEN 1
+      ELSE 0
+     END AS earliest_dec_suggestion_match
+    ,IF(participant_precedence = 1, 1, 0) AS unique_participant_counter
+  FROM
+   combined_data
+)
+
+-- Generate output
+SELECT 
+  * EXCEPT(participant_precedence)

```

Comment:
Where does participant_precedence get used in this query since we're filtering it out here?